### ISSUE TITLE
SyntaxError: Unexpected token <

### ISSUE THREAD

**Issue by lesterzone** [sqmk/node-chump#4]
Since the response from pushover is this one:

``` html
<!DOCTYPE html>
<html>
<head>
  <title>Pushover: Simple Mobile Notifications</title>
  <link rel="shortcut icon" href="https://pushover.net/favicon.ico" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link href="https://pushover.net/assets/application.css" media="all" rel="stylesheet" type="text/css" />
  <script src="https://pushover.net/assets/application.js" type="text/javascript"></script>
</head>
<body>
  <div id="whitey" class="">
    <div class="navbar navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <a class="brand" href="https://pushover.net/" style="width: 83px; height: 18px;">
            <img alt="Pushover" height="18"
                src="https://pushover.net/assets/writing-header.png"
                style="margin-bottom: -1px;" width="83" />
          </a>
        </div>
      </div>
    </div>

    <div class="container" style="min-height: 400px;">
  <div class="row">
    <div class="span12">
      <h2>Resource Not Found (or Incorrect Method)</h2>

      <p>
      The page or resource you requested could not be found.
      </p>
      <p>
      If you are attempting an <a href="https://pushover.net/api">API</a> call,
      verify the URL and the HTTP method (e.g., retry as a POST request instead
      of GET).
      </p>
      <p>
      Please try navigating through our <a
      href="https://pushover.net/">homepage</a> or view our <a
      href="https://pushover.net/api">API documentation</a>.
      </p>
    </div>
  </div>
</div>

  </div>

  <div class="dark_footer">
    <div class="container" style="padding: 2em;">

      <div class="row">
        <div class="span6" id="trademark">
          Pushover is a trademark and product of
          <a href="http://superblock.net/">Superblock, LLC</a>.
        </div>
        <div class="span6" id="copyright">
          Copyright &copy; 2012-2014 Superblock, LLC.  All rights reserved.
        </div>
      </div>

    </div>
  </div>
</body>
</html>
```

This line https://github.com/sqmk/chump/blob/master/lib/Transport.js#L59 is not catching parse error

Or maybe I'm missing something. I'm using same example as describe in the README file and my credentials works

**Comment by sqmk** [comment]
Can you show me sample code of what you are attempting to do? Based on the above output, I can't tell what endpoint the client is calling.

I can put in some changes to catch the exception (as it appears any error from pushover is returning html, which is not ideal for an API), but would like to know which endpoint chump is hitting to return this issue.

**Comment by sqmk** [comment]
Nevermind. Testing calls to messages is returning an error. I'll investigate!

**Comment by sqmk** [comment]
Released Chump v1.1.0. This should resolve your issue!

Thanks for the report :+1:

**Comment by lesterzone** [comment]
Indeed, issue fixed. Thanks four your time and effort.
